openvdb_houdini/SOP_OpenVDB_Merge.cc

```diff
@@ -0,0 +1,296 @@
+            "nameandclass"      "Grid Name and Grid Class"
+            "classandtype"      "Grid Class and Value Type"
+        }
+    }
+    parm {
+        name    "usereferencevdb"
+        label   "Use First Grid as Reference VDB Only"
+        type    toggle
+        default { "0" }
+    }
+    parm {
+        name    "resample"
+        label   "Resample Grids"
+        type    ordinal
+        default { "first" }
+        menu {
+            "first"         "To Match First Grid"
+            "highestres"    "To Match Highest-Res Grid"
+            "lowestres"     "To Match Lowest-Res Grid"
+        }
+        joinnext
+    }
+    parm {
+        name    "resampleinterp"
+        label   "Interpolation"
+        type    ordinal
+        default { "linear" }
+        menu {
+            "point"     "Nearest"
+            "linear"    "Linear"
+            "quadratic" "Quadratic"
+        }
+    }
+    groupsimple {
+        name    "mergemodegroup"
+        label   "Merge Mode"
+
+        parm {
+            name    "fogmode"
+            label   "Fog Mode"
+            type    ordinal
+            default { "add" }
+            menu {
+                "none"          "None"
+                "add"           "Add"
+                "multiply"      "Multiply"
+                "maximum"       "Maximum"
+                "minimum"       "Minimum"
+                "topounion"     "Activity Union"
+                "topointersect" "Activity Intersection"
+            }
+            joinnext
+        }
+        parm {
+            name    "scalarmode"
+            label   "Scalar Mode"
+            type    ordinal
+            default { "add" }
+            menu {
+                "none"          "None"
+                "add"           "Add"
+                "multiply"      "Multiply"
+                "maximum"       "Maximum"
+                "minimum"       "Minimum"
+                "topounion"     "Activity Union"
+                "topointersect" "Activity Intersection"
+            }
+        }
+        parm {
+            name    "distancemode"
+            label   "Distance Mode"
+            type    ordinal
+            default { "sdfunion" }
+            menu {
+                "none"          "None"
+                "sdfunion"      "SDF Union"
+                "sdfintersect"  "SDF Intersect"
+                "topounion"     "Activity Union"
+                "topointersect" "Activity Intersection"
+            }
+            joinnext
+        }
+        parm {
+            name    "vectormode"
+            label   "Vector Mode"
+            type    ordinal
+            default { "add" }
+            menu {
+                "none"          "None"
+                "add"           "Add"
+                "multiply"      "Multiply"
+                "topounion"     "Activity Union"
+                "topointersect" "Activity Intersection"
+            }
+        }
+        parm {
+            name    "boolmode"
+            label   "Bool Mode"
```

Comment:
I prefer Boolean VDBs
op_bool vs op_boolean
should probably be determined by what the VDB class is referred to as a token.